man/systemd-repart.xml

```diff
@@ -387,6 +387,15 @@
         <command>systemd-repart</command> was executed.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--sector-size=</option><arg>BYTES</arg></term>
```

Comment:
Any reason why not support si units? `--sector-size=4K`

Reply:
Lazyness, `parse_sector_size()` didn't support it. Given that the only possible values are 512, 2048 and 4096, the added benefit seems rather low